extensions/eo/README.md

```diff
@@ -25,12 +25,11 @@ The exact metadata that would appear in a STAC Collection record will vary depen
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
-| eo:gsd           | number                   | **REQUIRED.** Ground Sample distance. The nominal distance between pixel centers available, in meters. |
+| eo:gsd           | number                   | **REQUIRED.** Ground Sample Distance. The nominal distance between pixel centers available, in meters. |
 | eo:platform      | string                   | **REQUIRED.** Unique name of the specific platform the instrument is attached to. For satellites this would be the name of the satellite (e.g., landsat-8, sentinel-2A), whereas for drones this would be a unique name for the drone. |
 | eo:constellation | string                   | Name of the constellation that the platform belongs to. See below for details. |
 | eo:instrument    | string                   | **REQUIRED.** Name of instrument or sensor used (e.g., MODIS, ASTER, OLI, Canon F-1). |
 | eo:bands         | [Band Object]            | **REQUIRED.** This is a list of the available bands where each item is a Band Object. |
-| eo:epsg          | integer\|null            | [EPSG code](http://www.epsg-registry.org/) of the datasource, `null` if no EPSG code. |
```

Comment:
the epsg id isn't specific to EO, so I feel a better place for it would be the `proj` extension.  Completely open to not removing this.

Reply:
Hmm, I don't feel it's a good idea to have two places that define such a field. Related discussions in #356